apax/md/function_transformations.py

```diff
@@ -6,22 +6,22 @@
 
 @dataclasses.dataclass
 class FunctionTransformation:
-    def apply(self, model, n_models):
+    def apply(self, model):
         raise NotImplementedError
 
 
 def make_biased_energy_force_fn(bias_fn):
     def biased_energy_force_fn(positions, Z, idx, box, offsets):
-        gamd_fn = jax.value_and_grad(bias_fn, has_aux=True)
+        bias_and_grad_fn = jax.value_and_grad(bias_fn, has_aux=True)
 
-        (E_bias, results), F_bias = gamd_fn(positions, Z, idx, box, offsets)
+        (E_bias, results), F_bias = bias_and_grad_fn(positions, Z, idx, box, offsets)
 
         if "energy_unbiased" not in results.keys():
             results["energy_unbiased"] = results["energy"]
             results["forces_unbiased"] = results["forces"]
 
         results["energy"] = results["energy"] + E_bias
-        results["forces"] = results["forces"] + F_bias
+        results["forces"] = results["forces"] - F_bias
```

Comment:
:eyes:

Reply:
Forces are the negative gradient :upside_down_face: